### ISSUE TITLE
Suggestion: Maximised viewport on widescreen mode

### ISSUE THREAD

**Issue by emileb** [bibendovsky/bstone#320]
Not sure what you think but I think it looks good when the viewport is maximised when in widescreen mode.

I did a very hacky implementation, but was a bit confused with all the various size variables so it's probably not the correct method, but it seems to work for me:
https://github.com/emileb/bstone/commit/8f8d1f6aebda599a740384fe50fad0c4dd8a38e9

Original:
![device-2021-01-03-134143](https://user-images.githubusercontent.com/671781/103480519-c513ac00-4dcc-11eb-8b71-58c068bd62f4.png)

Maximised:
![device-2021-01-03-134236](https://user-images.githubusercontent.com/671781/103480614-2cc9f700-4dcd-11eb-9b7c-372ce8c55dce.png)

**Comment by bibendovsky** [comment]
Unless you plan to add option to control the viewport's size it's better, for now, to leave this code as mobile-related.

**Comment by emileb** [comment]
> Unless you plan to add option to control the viewport's size it's better, for now, to leave this code as mobile-related.

Yep good idea, I'll add as an option and you can decide if it's suitable for upstream. Thanks again.
